src/app/qgsmaptoolrotatefeature.h

```diff
@@ -105,7 +109,7 @@ class APP_EXPORT QgsMapToolRotateFeature: public QgsMapToolEdit
     double mRotationOffset;
 
     QPoint mStPoint;
-    QgsVertexMarker *mAnchorPoint = nullptr;
+    std::unique_ptr<QgsVertexMarker> mAnchorPoint = nullptr;
```

Comment:
I don't think that a `unique_ptr` needs to be initialized here.

Reply:
thanks a lot, fixed